eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -59,6 +64,12 @@
     },
     {
       "StaticConfigs": {
+        "Win2019": {
+          "OSVmImage": "windows-2019",
+          "Pool": "Azure Pipelines",
+          "RunProxyTests": true,
+          "CMAKE_GENERATOR": "Visual Studio 16 2019"
+        },
         "Win2022": {
           "OSVmImage": "windows-2022",
           "Pool": "Azure Pipelines",
```

Comment:
This should use our pool `azsdk-pool-mms-win-2022-general` instead of `Azure Pipelines`